app/views/devise/sessions/mobile_number.html.haml

```diff
@@ -7,9 +7,9 @@
         %fieldset.govuk-fieldset
           %legend.govuk-fieldset__legend.govuk-fieldset__legend--l
             %h1.govuk-fieldset__heading
-              Verify your login
+              Enter your mobile number
 
-          = f.govuk_text_field :otp_attempt, label: { text: "Please enter your six-digit verification code" }
+          = f.govuk_text_field :mobile_number, label: { text: "Enter your mobile number" }, type: "tel", autocomplete: "tel"
```

Comment:
We need `autocomplete` there to comply with [WCAG 2.1 Level AA](https://www.w3.org/WAI/WCAG21/Understanding/identify-input-purpose.html), according to https://design-system.service.gov.uk/patterns/telephone-numbers/.